### PR TITLE
PT-387/Fix Webhook NPEs

### PR DESCRIPTION
**Pull request by takecare**
#### Problem

We've been experiencing some event forwarding failures in our webhook lambda, which are not showing up on Github's webhook delivery reports. Most I could see from the logs was an NPE.

#### Solution

Writing a test based on data that caused the NPE was the first step. From there I could easily find what was causing the issue: a typo in one of the action names, in our model 😓 

##### Test(s) added

Yes.

### CODE REVIEW

File: github-hooks-web-service/src/main/java/com/novoda/github/reports/web/hooks/model/GithubAction.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
     @SerializedName("deleted")
     DELETED("deleted"),
 
-    @SerializedName("repopened")
```
Comment (eduardb) [inline github-hooks-web-service/src/main/java/com/novoda/github/reports/web/hooks/model/GithubAction.java]: 😂

**Comment by takecare** [comment]
@eduardb fair enough, will be adding it soon

File: github-hooks-web-service/src/main/java/com/novoda/github/reports/web/hooks/model/GithubAction.java (hunk 1 above)
Comment (frapontillo) [inline github-hooks-web-service/src/main/java/com/novoda/github/reports/web/hooks/model/GithubAction.java]: 💩

File: github-hooks-web-service/src/test/resources/reopened_action.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        "type": "User"
+      },
+      "state": "open",
+      "comments": 2,
+      "created_at": "2016-08-09T08:58:23Z",
+      "updated_at": "2016-08-10T14:45:27Z",
+      "pull_request": {
+        "url": "https://api.github.com/repos/takecare/dump/pulls/5"
+      },
+      "body": "this is a pr that stefanhoth closed and now it's being rePopened!"
```
Comment (frapontillo) [inline github-hooks-web-service/src/test/resources/reopened_action.json]: 🤣

**Pull request merged**